pom.xml

```diff
@@ -109,6 +109,7 @@
         <phoenix1.version>4.14.3-HBase-1.4</phoenix1.version>
         <phoenix2.version>5.1.3</phoenix2.version>
         <postgresql.jdbc.version>42.3.9</postgresql.jdbc.version>
+        <postgis.jdbc.version>2.5.1</postgis.jdbc.version>
```

Comment:
what  purpose that  using postgis driver to replace the original postgresql driver ?